tests/Issues/AddNodeAfterNodeStmt/Source/AddNextNopRector.php

```diff
@@ -51,13 +41,9 @@ public function refactor(Node $node)
             )
         );
 
-        $this->nodesToAddCollector->addNodeAfterNode(
+        return [
```

Comment:
This test is on purpose for add node after node with inline html usages, so need to be reverted.